crates/extensions/c8y_config_manager/src/config.rs

```diff
@@ -84,8 +84,8 @@ impl ConfigManagerConfig {
         let data_dir = tedge_config.data.path.clone().into();
         let mqtt_host = tedge_config.mqtt.client.host.clone();
         let mqtt_port = tedge_config.mqtt.client.port.get();
-        let tedge_http_address = tedge_config.http.bind.address;
-        let tedge_http_port = tedge_config.http.bind.port;
+        let tedge_http_address = tedge_config.http.client.address;
```

Comment:
@Bravo555 We should use `host` instead of `address so that we're consistent with the other parts of the code.